google-cloud-compute-v1/samples/acceptance/default_values_test.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../default_values"
+require_relative "helper"
+
+require "google/cloud/storage"
+
+
+class ComputeDefaultValuesTest < Minitest::Test
+  TEST_PREFIX = "some-prefix"
+
+  def setup
+    storage_client = ::Google::Cloud::Storage.new project_id: project
+    @bucket = storage_client.create_bucket random_bucket_name
+  end
+
+  def teardown
+    @bucket.delete
+  end
+
+  def test_set_usage_export_bucket_default
+    assert_output(/default prefix of `usage_gce`/) do
+      set_usage_export_bucket project: project, bucket_name: @bucket.name
+    end
+    sleep 5 # To make sure the settings are properly updated
```

Comment:
So even wait_until_done is not enough?

Reply:
I followed the python example and didn't test it by myself. I can remove it and see if it works.

Reply:
I tried running it without the sleeps locally and it seems like working fine.